src/main/java/org/jvnet/hudson/update_center/Plugin.java

```diff
@@ -331,29 +269,18 @@ public JSONObject toJSON() throws IOException {
             json.put("previousTimestamp", fisheyeDateFormatter.format(previous.getTimestamp()));
         }
 
-        json.put("title", getTitle());
-        if (page!=null) {
-            json.put("wiki",page.getUrl());
-            String excerpt = getExcerptInHTML();
-            if (excerpt!=null)
-                json.put("excerpt",excerpt);
-            String[] labelList = getLabels();
-            if (labelList!=null)
-                json.put("labels",labelList);
-        }
+        json.put("title", getName());
         String scm = getScmHost();
         if (scm!=null) {
             json.put("scm", scm);
         }
 
-        if (!json.has("excerpt")) {
-            // fall back to <description>, which is plain text but still better than nothing.
-            if(pom != null)
-            {
-                String description = plainText2html(selectSingleValue(pom, "/project/description"));
-                if (description!=null)
-                    json.put("excerpt",description);
-            }
+        json.put("wiki", "https://plugins.jenkins.io/" + artifactId);
```

Comment:
This results URLs even of private plugins get https://plugins.jenkins.io/... .
I added fab26b4e7afb7520690fd3024329989bade0cbb6 to have "-nowiki" switch disabled it and use the URL in pom.xml instead.